e2e/frontend/configEditor.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@grafana/plugin-e2e';
+import { formatExpectError } from './errors';
+
+const SENTRY_ORG_SLUG = 'mock-org-slug';
+const SENTRY_AUTH_TOKEN = 'mock-token';
+
+test.describe('Test config editor', () => {
+  test('empty configuration should throw valid error', async ({ createDataSourceConfigPage, page }) => {
+    const configPage = await createDataSourceConfigPage({ type: 'grafana-sentry-datasource' });
+
+    await expect(configPage.saveAndTest()).not.toBeOK();
+    await expect(page.getByTestId('data-testid Alert error')).toHaveText('invalid or empty organization slug');
+    
+    // await page.getByTestId('Data source settings page Delete button').click();
```

Comment:
I think it's fine to either include this commented code or remove it entirely, your call :)